src/app/find-properties/[[...opa_id]]/page.tsx

```diff
@@ -33,6 +33,7 @@ const MapPage = ({ params }: MapPageProps) => {
   const [featureCount, setFeatureCount] = useState<number>(0);
   const [currentView, setCurrentView] = useState<BarClickOptions>("detail");
   const [loading, setLoading] = useState(true);
+  const [loadingError, setLoadingError] = useState(false);
```

Comment:
Can we rename `loadingError` and its setter to something like `hasLoadingError` and `setHasLoadingError`?

This variable has a boolean value, but its current name can give the wrong impression. `loadingError` sounds like a name for something else, such as an error response object.

Reply:
This variable has a boolean value, but its current name can give the wrong impression. `loadingError` sounds like a name for something else, such as an error response object. (I realize the line above has a boolean variable named `const loading`, but let's not follow that example.)